### PR TITLE
Pom/node version

### PR DESCRIPTION
**Pull request by seanfeld**
# Fixit update node version 

## Description
Update node version because github actions build allows us to move off bamboo and node 10

## PR Type
- [ ] Bug Fix
- [ ] Feature
- [ ] Build Fix
- [ ] Testing
- [x] General Improvement
- [ ] Cherry Pick

### CODE REVIEW

**Comment by mrahanjam** [comment]
We probably wanna update node version in the workflow as well:
https://github.com/cdapio/cdap-ui/blob/6d349e236c3d58b77b660168508597a2b7e35605/.github/workflows/build.yml#L44

**Review by mrahanjam — APPROVED** [review]
LGTM, left a note on updating github workflow

**Comment by seanfeld** [comment]
> We probably wanna update node version in the workflow as well:
> 
> https://github.com/cdapio/cdap-ui/blob/6d349e236c3d58b77b660168508597a2b7e35605/.github/workflows/build.yml#L44

good point I wasn't very aware of that file.

File: pom.xml [hunk 1]
```diff
@@ -368,7 +368,7 @@
                   <goal>install-node-and-yarn</goal>
                 </goals>
                 <configuration>
-                  <nodeVersion>v10.16.2</nodeVersion>
+                  <nodeVersion>v16.16.0</nodeVersion>
                   <yarnVersion>v1.7.0</yarnVersion>
```
Comment (mrahanjam) [inline pom.xml:372]: Don't we wanna update `yarn` as well?

Comment (seanfeld) [reply pom.xml:372]: no that should be a separate pr

**Review by itsankit-google — APPROVED** [review]

**Comment by sumengwang** [comment]
Looks like some e2e tests are failing. Is there any concern?

**Pull request merged**